Source/Core/Runtime/Properties/LockablePropertyReference.cs

```diff
@@ -37,15 +36,15 @@ public LockablePropertyReference()
 
         public LockablePropertyReference(LockableProperty property)
         {
-            Target = new SceneObjectReference(property.SceneObject.UniqueName);
+            Target = new SingleSceneObjectReference(property.SceneObject.Guid);
             Type = property.GetType().AssemblyQualifiedName;
         }
 
-        public LockablePropertyReference(string sceneObjectName, Type type)
-        {
-            Target = new SceneObjectReference(sceneObjectName);
-            Type = type.AssemblyQualifiedName;
-        }
+        //public LockablePropertyReference(string sceneObjectName, Type type)
+        //{
+        //    Target = new SceneObjectReference(sceneObjectName);
+        //    Type = type.AssemblyQualifiedName;
+        //}
```

Comment:
There is another constructor that takes a property instead of a unique name, this was probably used for tests or not at all. I deleted it instead of just commenting it out.